src/read/elf/relocation.rs

```diff
@@ -3,13 +3,13 @@ use alloc::vec::Vec;
 use core::fmt::Debug;
 use core::slice;
 
-use crate::elf;
 use crate::endian::{self, Endianness};
 use crate::pod::Pod;
 use crate::read::{
     self, Error, ReadRef, Relocation, RelocationEncoding, RelocationFlags, RelocationKind,
     RelocationTarget, SectionIndex, SymbolIndex,
 };
+use crate::{elf, Bytes, Endian, I32, I64, U32, U64};
```

Comment:
Leave `use crate::elf;` how it was. Move `Bytes` to `use crate::read`. Move the rest to `use::crate::endian`.